site2/website-next/src/theme/DocVersionBanner/index.js

```diff
@@ -41,7 +41,7 @@ function UnmaintainedVersionLabel({ siteTitle, versionMetadata }) {
       }}
     >
       {
-        "This is documentation for {siteTitle} {versionLabel}, which is no longer actively maintained."
+        "This documentation is for an out-of-date {siteTitle} {versionLabel} version of Apache Pulsar. "
```

Comment:
```suggestion
        "This documentation is for an out-of-date version of Apache Pulsar. "
```

Ditto. The sidebar and URL should already identify which version it is.